docs/source/settings.rst

```diff
@@ -123,13 +123,20 @@ A string referring to one of the following bundled classes:
 * ``gevent``   - Requires gevent >= 0.13
 * ``tornado``  - Requires tornado >= 0.2
 * ``gthread``  - Python 2 requires the futures package to be installed
-* ``gaiohttp`` - Requires Python 3.4 and aiohttp >= 0.21.5
+* ``gaiohttp`` - Deprecated. Please use
```

Comment:
we should just mark it as deprecated. documentation is already here to point to external asyncio workers

Reply:
man, i just started using it this week! :)